q2_feature_table/plugin_setup.py

```diff
@@ -235,12 +243,18 @@
                         'be observed in to be retained. If no value is '
                         'provided this will default to infinity (i.e., no '
                         'maximum sample filter will be applied).'),
-        'metadata': 'Feature metadata used in conjuction with `where` '
-                    'parameter to select features to retain.',
+        'metadata': 'Sample metadata used with `where` parameter when '
```

Comment:
`filter_features` filters by feature metadata, not sample metadata --- can you please update this text to match that?